crates/base/src/commands.rs

```diff
@@ -32,3 +35,20 @@ pub async fn start_server(
     .await?;
     server.listen().await
 }
+
+fn set_v8_flags() {
```

Comment:
Is this a refactor? If so, can you create a PR just for this?

Reply:
@laktek Yeah, It had to be changed due to my recent PKU commit. Because the commit changes initialize the v8 platform explicitly, I had to pass the v8 flags before initializing it since the v8 will complain and crash about trying to change v8 flags already marked as immutable😋

I will submit the PR just for this right now!